.github/workflows/documentation.yml

```diff
@@ -0,0 +1,38 @@
+name: Do docs need to be updated?
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: changelog-check
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      
+      - name: Get info on if image definition has changed
+        id: changed-files-image-def
+        uses: tj-actions/changed-files@v32
+        with:
+          files: |
+            internal/imagedefinition/**
```

Comment:
Maybe at some point we should actually look only at the image definition file itself changing, ignoring changes to the _test file.

Reply:
I thought about this but went with `**` in case new files get added in that directory. I thought those would likely require docs updates.